README.md

```diff
@@ -6,13 +6,19 @@
 
 ## Overview
 
-With the [`synthetics-test-automation-bitrise-step-run-tests` step][1], you can run Synthetic tests during your Bitrise CI, ensuring that all your teams using Bitrise can benefit from Synthetic tests at every stage of the software lifecycle. This step uses the [Datadog CI Synthetics command][2].
+With the `synthetics-test-automation-bitrise-step-run-tests` step, you can run Synthetic tests during your Bitrise CI, ensuring that all your teams using Bitrise can benefit from Synthetic tests at every stage of the software lifecycle. This step uses the [Datadog CI Synthetics command][2].
 
 ## Setup
 
+This step is not available on the official Bitrise Step Library.
 To get started:
 
-1. Add this step to your workflow. You can also configure it locally by referencing this step in your `bitrise.yml` file. For more information, see the [official Bitrise documentation][3].
+1. Add the following git URL to your workflow. See the [official Bitrise documentation][3] on how to do that though the Bitrise app. You can also configure it locally by referencing the git URL in your `bitrise.yml` file.
+
+```yml
+- git::https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests.git:
```

Comment:
Can you give examples with pinned tags, like in the official documentation?

Those tags should be updated by the `.github/workflows/release-version.yml` action ([source](https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests/blob/ffcf0a96b46c9a75d22dfb1bf8f9aaef45f8d8d9/.github/workflows/release-version.yml#L51-L52)).